core/node/api_server/src/web3/namespaces/en.rs

```diff
@@ -147,27 +147,28 @@ impl EnNamespace {
     }
 
     #[tracing::instrument(skip(self))]
-    pub async fn get_ecosystem_contracts_impl(&self) -> Result<EcosystemContracts, Web3Error> {
-        Ok(self
-            .state
-            .api_config
-            .l1_bridgehub_proxy_addr
-            .map(|bridgehub_proxy_addr| EcosystemContracts {
-                bridgehub_proxy_addr,
-                state_transition_proxy_addr: self
-                    .state
-                    .api_config
-                    .l1_state_transition_proxy_addr
-                    .unwrap(),
-                transparent_proxy_admin_addr: self
-                    .state
-                    .api_config
-                    .l1_transparent_proxy_admin_addr
-                    .unwrap(),
-                l1_bytecodes_supplier_addr: self.state.api_config.l1_bytecodes_supplier_addr,
-                l1_wrapped_base_token_store: self.state.api_config.l1_wrapped_base_token_store,
-            })
-            .context("Shared bridge doesn't supported")?)
+    pub async fn get_l1_ecosystem_contracts_impl(&self) -> Result<EcosystemContracts, Web3Error> {
+        Ok(EcosystemContracts {
+            bridgehub_proxy_addr: self
+                .state
+                .api_config
+                .l1_ecosystem_contracts
+                .bridgehub_proxy_addr
+                .unwrap(),
+            state_transition_proxy_addr: self
+                .state
+                .api_config
+                .l1_ecosystem_contracts
+                .state_transition_proxy_addr,
+            transparent_proxy_admin_addr: None,
```

Comment:
btw this looks like a breaking change. I see that you removed `transparent_proxy_admin_addr` from EN but we can't be sure that someone external doesn't uses this field. I don't think you can just drop it.

Reply:
it's completely useless field for everyone